HW_3/hw03.Rmd

```diff
@@ -0,0 +1,170 @@
+# Homework 03
+
+You will analyze data looking at the relationship between **green reputation** and three personality traits--**compassion**, **intellectual curiosity**, and **openness to experiences**.
+The dataset includes data from **students** and **non-students**.
+
+```{r}
+dictionary <- readr::read_csv("green_dictionary.csv")
+green_data <- readr::read_csv("green_data.csv")
+library(dplyr) 
+library(ggplot2)
+library(tidyverse)
+``` 
+
+For your assignment, do the following.
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+```{r}
+summary(green_data) 
+
+```
+  
+ 
+  2. Compute total scores for the four scales. 
+     Recode variables as needed.
+```{r}
+green_data %>%
+mutate(
+  across(eeverything(),
+ ~ recode(.x , "-99" = NA, .default = .x)
+))
+
+#   ~ says that you are going to use the .x symbol , .x marks the spot 
+# 'everything()' = ALL COLUMNS 
+# .missing is for when you look for missing values 
+# NA is for when this is missing 
+# .x = the current column 
+
+```   
+  
+  
+  3. Rescale the variables so that they go from 0-100 instead of the original range.
+     Name the rescaled variables `*_pomp`.
+```{r}
+reversed <- c("green1", "green2","green3", "green4", 
+              "green5" , "comp1", "comp2", "comp3", 
+              "comp4", "comp5", "comp6", "comp7", "comp8", "comp9", "comp10", 
+              "intel1", "intel2", "intel3", "intel4", "intel5", "intel6",
+              "intel7", "intel8", "intel9", "intel10", "open1",
+              "open2", "open3", "open4", "open5", "open6", "open7", "open8",
+              "open9", "open10")
+new_green_data <- green_data %>% 
+                    mutate(
+                           across(all_of(reversed),
+                           ~ recode(.x, "1" = 0, "2" = 25, "4" = 75, "5" = 100)
+                          )) %>% 
+  rowwise() %>% 
+  mutate(green_pomp = sum(c
+                         (green1,green2,green3,green4,green5), na.rm = TRUE),
+         comp_pomp = sum(c
+                        (comp1,comp2,comp3,comp4,comp5,comp6,comp7,comp8,comp9,comp10), na.rm = TRUE),
+         intel_pomp = sum(c
+                         (intel1,intel2,intel3,intel4,intel5,intel6,intel7,intel8,intel9,intel10), na.rm = TRUE),
+         open_pomp = sum(c
+                        (open1,open2,open3,open4,open5,open6,open7,open8,open9,open10), na.rm = TRUE),) 
```

Comment:
The values are so large on the plots because you recoded 1,2,3,4,5 to be 0,25,50,75,100 respectively and then summed all of those, so the values are going to be really big.

1) You should use the mean function, rather than sum.

2) It also looks like you only recoded (i.e., changed 1,2,3,4,5 to 0,25,50,75,100) the reverse-coded items? You could change the non-reverse-coded items on the same scale to be 0-100 and then use the mean function when you create the total scores.

Reply:
Thank you! I was wondering why my numbers were so big :)